protos/common/window.proto

```diff
@@ -0,0 +1,24 @@
+syntax = "proto3";
+
+package common;
+
+import "google/protobuf/timestamp.proto";
+
+// Sets the golang package for the protobuf generated code
+option go_package = "github.com/opencost/opencost/core/pkg/common/pb";
+
+// Resolution represents the time granularity for data aggregation
+enum Resolution {
+  RESOLUTION_UNSPECIFIED = 0;
```

Comment:
Not sure if this is standard practice but I don't think we want to be able to let this be unspecified